recipe/meta.yaml

```diff
@@ -10,44 +10,49 @@ source:
   sha256: 793359e94eb436b34877b4305570774f9b1bba34c4671238512b80b60456d9ce
 
 build:
+  number: 0
+  # maturin isn't available on s390x
+  skip: true  # [py<39 or s390x]
   script:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
-    - {{ PYTHON }} -m pip install . -vv
-  number: 1
-  skip: true  # [py<39]
-  skip: true  # [ppc64le and python_impl == 'pypy']
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - maturin >=0.14,<0.15
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - cargo-bundle-licenses
+    - cargo-bundle-licenses  # [win]
```

Comment:
They are already in `host` but for `win` I put them again because of failure if they aren't available. Something is wrong internally in conda-build if it builds rust packages. I saw the same issue here too https://github.com/AnacondaRecipes/watchfiles-feedstock/blob/bd34ea8d592c76ec9a4f96f8096a5103ecdfd914/recipe/meta.yaml#L23-L35

Reply:
See https://github.com/AnacondaRecipes/lazrs-python-feedstock/pull/1#discussion_r1206973960